Assets/scss/components/_header.scss

```diff
@@ -42,17 +46,40 @@
 
         &_menu {
             flex: 1;
-            order: 2;
+            margin-left: 6%;
```

Comment:
I need to ask my CSS experts about using percentages. This is the second time I've seen them used in a PR from this group. Maybe I'm not up-to-date on that.

Reply:
So, here I have given 6% of margin to left. So that it will maintained that margin while increasing the size of the screen. This is 6% will get calculated from its parent width.